GVRf/Framework/src/org/gearvrf/GVRRenderData.java

```diff
@@ -261,7 +279,12 @@ public boolean isDone() {
             return mFutureMesh.isDone();
         }
     }
-
+    
+    public void addPass(GVRMaterial material, int cullFace) {
+    	NativeRenderData.addPass(getNative(), material.getNative(), cullFace);
+    }
+    
+    
     /**
```

Comment:
Imho, I think you may need a getter which return a list of RenderPass. And also, for better maintenance, you may need a java class for RenderPass. Currently it is a little bit difficult for developer to know exactly how many renderpasses are in a RenderData instance.